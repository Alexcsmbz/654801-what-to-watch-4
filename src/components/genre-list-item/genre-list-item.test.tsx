import GenreListItem from './genre-list-item';

describe(`GenreListItem snapshot test`, () => {
  it(`GenreListItem should render genre`, () => {
    const tree = renderer.create(
        <GenreListItem
          item={`genre`}
          onClick={() => {}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
