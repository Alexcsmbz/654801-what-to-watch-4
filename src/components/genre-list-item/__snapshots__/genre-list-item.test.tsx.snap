// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreListItem snapshot test GenreListItem should render genre 1`] = `
<li
  className="catalog__genres-item undefined"
  onClick={[Function]}
>
  <div
    className="catalog__genres-link"
  >
    genre
  </div>
</li>
`;
